__tests__/pages/webhooks/order-calculate-taxes.test.ts

```diff
@@ -127,8 +128,40 @@ describe("api/webhooks/order-calculate-taxes", () => {
     expect(data.shipping_tax_rate).toBe("0.23");
     expect(data.lines.length).toBe(1);
     expect(data.lines[0].total_gross_amount).toBe("34.44");
-    expect(data.lines[0].total_net_amount).toBe("28");
+    expect(data.lines[0].total_net_amount).toBe("28.00");
     expect(data.lines[0].tax_rate).toBe("0.23");
     expect(res.statusCode).toBe(200);
   });
+
+  it("propagates discounts over lines", async () => {
+    const mockedTaxJarResponseData = dummyFetchTaxesResponse;
+    const mockedTaxJarResponse = mockedFetchTaxes.mockImplementationOnce(() => {
+      return mockedTaxJarResponseData;
+    });
+    const { req, res } = mockRequest({
+      method: "POST",
+      event: "order_calculate_taxes",
+      domain: "example.com",
+    });
+
+    const orderPayload = dummyOrderPayload;
+    orderPayload.discounts = [{amount: "2"}, {amount: "1"}]
+    const linePayload = orderPayload.lines[0]
+    orderPayload.lines = [linePayload, linePayload]
+
+    req.body = [orderPayload];
+
+    // @ts-ignore
```

Comment:
Same as in prev comment - please add explanation why `is-ignore` is used here
```suggestion
    // @ts-expect-error
```